app/experimenter/nimbus-ui/src/components/FormBranches/FormBranch.tsx

```diff
@@ -0,0 +1,228 @@
+              onChange={handleChange("description")}
+            />
+          </Form.Group>
+          <Form.Group as={Col} controlId={`${id}-ratio`} sm={2} md={2}>
+            <Form.Label>Ratio</Form.Label>
+            {equalRatio ? (
+              <p data-testid="equal-ratio" className="p-0 m-0">
+                Equal
+              </p>
+            ) : (
+              <Form.Control
+                type="text"
+                defaultValue={ratio}
+                onChange={handleChange("ratio")}
+              />
+            )}
+          </Form.Group>
+          <Form.Group as={Col} sm={1} className="align-top text-right">
+            {!isReference && onRemove && (
+              <Button
+                data-testid="remove-branch"
+                variant="light"
+                className="bg-transparent border-0 p-0 m-0"
+                title="Remove branch"
+                onClick={handleRemoveClick}
+              >
+                <DeleteIcon width="18" height="18" />
+              </Button>
+            )}
+          </Form.Group>
+        </Form.Row>
+      </Form.Group>
+
+      {experimentFeatureConfig === null ? (
+        <Form.Group className="px-2 mx-3">
+          <Form.Row>
+            <Button
+              variant="outline-primary"
+              size="sm"
+              data-testid="feature-config-add"
+              onClick={onAddFeatureConfig}
+            >
+              + Feature configuration
+            </Button>
+          </Form.Row>
+        </Form.Group>
+      ) : (
+        <Form.Group
+          className="px-3 py-2 border-top"
+          data-testid="feature-config-edit"
+        >
+          <Form.Row>
+            <Col className="px-2">Feature configuration</Col>
+            <Form.Group as={Col} sm={1} className="align-top text-right">
+              <Button
+                variant="light"
+                className="bg-transparent border-0 p-0 m-0"
+                data-testid="feature-config-remove"
+                title="Remove feature configuration"
+                onClick={onRemoveFeatureConfig}
+              >
+                <DeleteIcon width="18" height="18" />
+              </Button>
+            </Form.Group>
+          </Form.Row>
+          <Form.Row className="align-middle">
+            <Form.Group as={Col} controlId={`${id}-feature`}>
+              <Form.Control
+                as="select"
+                data-testid="feature-config-select"
+                onChange={handleFeatureConfigChange}
+                value={featureConfig!.findIndex(
+                  (feature) =>
+                    experimentFeatureConfig !== null &&
+                    feature !== null &&
+                    feature.slug === experimentFeatureConfig.slug,
+                )}
+              >
+                <option value="">Select...</option>
+                {featureConfig?.map(
+                  (feature, idx) =>
+                    feature !== null && (
```

Comment:
Could this be `feature && (`?

Reply:
I want to say I *did* have `feature &&` here and Typescript didn't like it